example/example-node.js

```diff
@@ -47,7 +47,7 @@ const htmlString = `<!DOCTYPE html>
                         <ol>
                             <li>Srilankan Tea
```

Comment:
```suggestion
                            <li>Srilankan <strong>Tea</strong>
```
Just to cover the formatted and nested list contents scenario.

Reply:
makes sense. will add it.